sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -95,7 +97,7 @@ const defaults: FormListing = {
   unitsAvailable: 0,
   unitAmenities: "",
   servicesOffered: "",
-  yearBuilt: 0,
+  yearBuilt: 2021,
```

Comment:
Maybe generate year dynamically based on the current date?

Reply:
This is just temporary through the MVP :)